nix/llvm-lit.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub,
+  python3,
+  stdenv, cmake, ninja, makeWrapper }:
+let
+  src = fetchFromGitHub {
+    owner = "llvm";
+    repo = "llvm-project";
+    rev = "llvmorg-17.0.3";
+    hash = "sha256-fXkYSYwhMeY3nLuuSlO2b/TpTxa+61zxmsfFg7a2Gbo=";
+  };
+
+  llvm-lit = python3.pkgs.buildPythonPackage {
+    inherit src;
+
+    pname = "llvm-lit";
+    version = "17.0.3";
+    format = "setuptools";
+    sourceRoot = "source/llvm/utils/lit";
+    propagatedBuildInputs = [
+      python3.pkgs.setuptools
```

Comment:
I forget the rules for this for python (diff than normal packages :sweat: ) but offhand why propagate setuptools, for use of this as a build-dep of some other python thing?

Reply:
Yes, this is my mistake. I should packaged it as an application not a library.